src/main/java/org/frc5687/rapidreact/OI.java

```diff
@@ -1,64 +1,40 @@
-/* Team 5687 (C)2020-2021 */
+/** Team 5687 (C)2021-2022
+ * Joystick and gamepad control for the robot.
+ * Also has button inits.
+ * Has some instructions on how to switch controls.
+*/
 package org.frc5687.rapidreact;
 
 import static org.frc5687.rapidreact.util.Helpers.*;
-
 import edu.wpi.first.wpilibj.Joystick;
-import edu.wpi.first.wpilibj2.command.button.Button;
-import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import org.frc5687.rapidreact.subsystems.DriveTrain;
-import org.frc5687.rapidreact.util.AxisButton;
 import org.frc5687.rapidreact.util.Gamepad;
 import org.frc5687.rapidreact.util.OutliersProxy;
 
 public class OI extends OutliersProxy {
-    protected Gamepad _driverGamepad;
-    protected Joystick _leftJoystick;
-    protected Joystick _rightJoystick;
-
-    protected Button _driverRightStickButton;
-
-    private JoystickButton _trigger;
-    private JoystickButton _thumbButton;
-    private JoystickButton _shootButton;
-    private JoystickButton _resetYawButton;
-
-    private Button _driverAButton;
-    private Button _driverBButton;
-    private Button _driverXButton;
-    private Button _driverYButton;
-    private Button _driverRightTrigger;
+    protected Gamepad Gamepad;
+    protected Joystick Rotation;
+    protected Joystick Translation;
 
```

Comment:
Since these are all member variables, they should be camelcase starting with an underscore.
IE, 
`private Gamepad _gamepad;`
They should also be private, although it's a little less critical here since we don't anticipate subclasses of OI.
[See java naming conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)